auction_keeper/gas.py

```diff
@@ -73,7 +73,7 @@ def get_gas_price(self, time_elapsed: int) -> Optional[int]:
             initial_price = int(round(fast_price * self.initial_multiplier))
 
         return GeometricGasPrice(initial_price=initial_price,
-                                 every_secs=30,
+                                 every_secs=42,
```

Comment:
Again, we don't want this configurable to constrain config complexity and prevent inappropriate values.

Reply:
Was actually on the fence with 56.  A smaller initial adjustment seemed prudent.